README.md

```diff
@@ -1,34 +1,18 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Atom Linter](https://github.com/AtomLinter/Linter) providing an interface to [HTMLHint](https://github.com/yaniswang/HTMLHint). It will be
```

Comment:
The correct URL is <https://github.com/atom-community/linter>, also it's just "Linter".